src/Public/Redis.ps1

```diff
@@ -109,11 +131,14 @@ function Invoke-RedisScript
 
         [Parameter()]
         [object[]]
-        $Arguments
+        $Arguments,
+
+        [Parameter()]
+        $ConnectionName
```

Comment:
Missing `[string]` type for parameter (plus the rest as well - save spam 😂)

Reply:
Done